src/pages/home/base/assurance/api-reference.md

```diff
@@ -73,3 +73,21 @@ iOS
 Flutter
 
 <Tabs query="platform=flutter&api=start-session"/> --->
+
+## startSession (Quick Connect)
+
+Displays a Quick Connect authorization overlay to begin an AEP Assurance session.
```

Comment:
"... to begin an Experience Platform Assurance session." - since AEP is against branding guidelines.